src/libsystemd-network/sd-dhcp-server.c

```diff
@@ -712,7 +715,54 @@ static int get_pool_offset(sd_dhcp_server *server, be32_t requested_ip) {
         return be32toh(requested_ip & ~server->netmask) - server->pool_offset;
 }
 
-static int dhcp_server_relay_message(sd_dhcp_server *server, DHCPMessage *message, size_t opt_length) {
+static size_t relay_option_length(sd_dhcp_server *server) {
+        size_t length = 0;
+        if (server->agent_circuit_id)
+                length += strlen(server->agent_circuit_id) + 2;
+        if (server->agent_remote_id)
+                length += strlen(server->agent_remote_id) + 2;
+
+        return length;
+}
+
+static uint8_t* append_suboption(uint8_t* dst, uint8_t subcode, char* value, uint8_t length) {
```

Comment:
- Please move `*` right before the variable name.
- Please add `const` for `value`.

`uint8_t *append_suboption(uint8_t *dst, uint8_t subcode, const char *value, uint8_t length)`